website/docs/d/virtual_network_gateway_connection.html.markdown

```diff
@@ -72,6 +72,10 @@ output "virtual_network_gateway_connection_id" {
     Only a single policy can be defined for a connection. For details on
     custom policies refer to [the relevant section in the Azure documentation](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-ipsecikepolicy-rm-powershell).
 
+* `traffic_selector_policy` (Optional) A `traffic_selector_policy` which allows to specify traffic selector policy proposal to be used in a virtual network gateway connection.
+    Only one block can be defined for a connection.
+    For details about traffic selectors refer to [the relevant section in the Azure documentation](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-connect-multiple-policybased-rm-ps).
```

Comment:
can we remove "optional" from this, since this a read-only value (in the Data Source) - the reference in `ipsec_policy` above is a bug in the documentation unfortunately

could we also add this to [the Resource documentation too](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/website/docs/r/virtual_network_gateway_connection.html.markdown)?